### ISSUE TITLE
Pre-SoPN ballot text

### ISSUE THREAD

**Issue by pmk01** [DemocracyClub/WhoCanIVoteFor#1147]
This text (from candidates) is pretty clear. Maybe we should show it on WCIVF too.

![Screenshot 2022-04-07 204233](https://user-images.githubusercontent.com/61510609/162283546-1f78aa7d-0238-4315-af05-dfb861e7eaf9.png)

**Comment by pmk01** [comment]
Further thought on this: IIRC we currently say something like ' we know of at least n candidates'. This text should be modified in light of the above. Perhaps the full text could read:

> We know of [n] candidates who have expressed intention to stand in this election. However, these candidates haven't yet been confirmed by the official 'Statement of Persons Nominated' from the council. This means they might not all end up on the ballot paper.
> 
> We will manually verify each candidate when the official statement is published.

Does that sound ok? too wordy? A case where we need to balance the legal language with what people actually need to know.
(possibly we could do with an explanation of all this somewhere we could link to)

**Comment by VirginiaDooley** [comment]
Having discussed with @pmk01, we have settled on: 

> These candidates will not be confirmed until the council publishes the "official candidate list” on XX/XX/XXXX. Once nomination papers are published, we will manually verify each candidate.
